soc/riscv/nordic_nrf/nrf54h/Kconfig.defconfig.series

```
@@ -0,0 +1,18 @@
+# Copyright (c) 2024 Nordic Semiconductor ASA
+# SPDX-License-Identifier: Apache-2.0
+
+if SOC_SERIES_NRF54HX
+
+rsource "Kconfig.defconfig.nrf54h*"
+
+config SOC_SERIES
+	default "nrf54h"
+
+DT_CHOSEN_Z_SRAM = zephyr,sram
+DT_CHOSEN_Z_CODE = zephyr,code-partition
+
+config BUILD_OUTPUT_ADJUST_LMA
+	depends on !XIP
+	default "$(dt_chosen_partition_addr_hex,$(DT_CHOSEN_Z_CODE))-$(dt_chosen_reg_addr_hex,$(DT_CHOSEN_Z_SRAM))"
```

Just a note, I think we could also go with:
```suggestion
	default "$(dt_chosen_reg_addr_hex,$(DT_CHOSEN_Z_FLASH)) + \
		 $(dt_chosen_reg_addr_hex,$(DT_CHOSEN_Z_CODE)) - \
		 $(dt_chosen_reg_addr_hex,$(DT_CHOSEN_Z_SRAM))"
```
where `DT_CHOSEN_Z_FLASH = zephyr,flash`.

---

Yes, that's right, even though I prefer the new helper as the gparent relationship is captured internally.